lib/set.js

```diff
@@ -1,5 +1,15 @@
-var set = function(data) {
-  this.data = data;
+var _ = require('lodash');
+
+var set = function(path, data) {
+  if (!data){
+    // Update delete  path
```

Comment:
I don't think this will be our delete condition.

My thought process was, that if you update any item, the old detail should be treated as deleted.
Did you had anything else in mind?

Reply:
Yes, so basically updated with anything, previous value is deleted for us.
So it should be `!data` check. Since it will just update the value to `undefined`
And the previous value will become deleted for us.
Even if you pass some value in place of `undefined` after update, previous value is deleted for us.
I think we should not have this condition